packages/libs/react-ui/src/components/Form/TextField/TextField.tsx

```diff
@@ -41,6 +46,10 @@ export interface ITextFieldProps extends PickedAriaTextFieldProps {
   endAddon?: ReactNode;
   isOutlined?: boolean;
   inputFont?: 'body' | 'code';
+  /*
+   * using native the definition from react for type but blocking all other types that we dont want (specifically blocking number)
+   */
+  type?: 'text' | 'search' | 'url' | 'tel' | 'email' | 'password';
```

Comment:
Based on react-aria's implementation I think it's better to not change the types on this component